README.md

```diff
@@ -1,229 +1,124 @@
-
-1. Select **Create**. The new user flow appears in the **User flows** list. If necessary, refresh the page.
-
-## Associate the  app with the user flow
-
-Although many applications can be associated with your user flow, a single application can only be associated with one user flow.
-
-To associate your user flow with your app, follow these steps.
-
-1. On the sidebar menu, select **Identity**.
-1. Select **External Identities**, then **User flows**.
-1. In the **User flows** page, select the **User flow name** you created earlier, for example, _SignInSignUpSample_.
-1. Under **Use**, select **Applications**.
-1. Select **Add application**.
-1. Select the application from the list such as _ciam-client-app_ or use the search box to find the application, and then select it.
+* <a href="https://developer.android.com/studio/archive" target="_blank">Android Studio Dolphin | 2021.3.1 Patch 1</a>.
+* Microsoft Entra External ID for customers tenant. If you don't already have one, <a href="https://aka.ms/ciam-free-trial?wt.mc_id=ciamcustomertenantfreetrial_linkclick_content_cnl" target="_blank">sign up for a free trial</a>.
 
-1. Choose **Select**.
+## Project setup
 
-## Clone sample Android mobile application
+To enable your application to authenicate users with Microsoft Entra, Microsoft Entra ID for customers must be made aware of the application you create. The following steps show you how to:
 
-1. Open Terminal and navigate to a directory where you want to keep the code.
-1. Clone the application from GitHub by running the following command:
+### Step 1: Register an application
 
-   ```bash 
-   git clone https://github.com/AzureAD/msal-android-native-auth-sample-app-preview.git 
-   ``` 
+Register your app in the Microsoft Entra admin center using the steps in [Register an application](https://learn.microsoft.com/entra/external-id/customers/how-to-run-native-authentication-sample-android-app#register-an-application).
 
-## Configure the sample Android mobile application
+### Step 2: Enable public client and native authentication flows
 
-1. In Android Studio, open the project that you cloned.
-1. Add the following reference for GitHub credentials to your global `gradle.properties`.
+Enable public client and native authentication flows for the registered application using the steps in [Enable public client and native authentication flows](https://learn.microsoft.com/entra/external-id/customers/how-to-run-native-authentication-sample-android-app#enable-public-client-and-native-authentication-flows).
 
-    ```gradle 
-    GITHUB_USERNAME="<username>" 
-    GITHUB_TOKEN="<generated-token>" 
-    ``` 
+### Step 3: Grant API permissions
 
-   The default location of this global `gradle.properties` is:
+Grant API permissions to the registered application by following the steps in [Grant API permissions](https://learn.microsoft.com/entra/external-id/customers/how-to-run-native-authentication-sample-android-app#grant-api-permissions).
```

Comment:
@Yuki-YuXin @borgesmb are special API permissions (i.e. scopes) needed to call the protected web API? I can't remember exactly, but I think yes? 

If yes, could you give some guidance here so Henry can easily update it?

Reply:
The special API permissions are built up by developers themselves. So it's hard to give a fixed answer.
The tutorial is the core one for scopes configuration.
https://learn.microsoft.com/en-us/entra/identity-platform/quickstart-configure-app-expose-web-apis